lldb/unittests/Process/Utility/RegisterContextFreeBSDTest.cpp

```diff
@@ -362,7 +363,15 @@ TEST(RegisterContextFreeBSDTest, arm64) {
   EXPECT_GPR_ARM64(lr, lr);
   EXPECT_GPR_ARM64(sp, sp);
   EXPECT_GPR_ARM64(pc, elr);
+#if __FreeBSD_version >= 1400084
```

Comment:
This will be correct when the host running the debugger is the same as the target (or at least the same version) but we should handle this based on target version

Reply:
Good point. Though these tests are likely only run on the machine that built them, but if lldb is loading a FreeBSD corefile it would need to know there was a difference. That at least needs checking, even if it may appear to "just work".

Making cross version core files work is more than I have time for here, so if any FreeBSD person wants to do a proper version of this fix, feel free.

(I did look for a runtime way to get `__FreeBSD_version` but none of the version strings I found contained the final part of the macro's value)